README.md

```diff
@@ -233,10 +233,10 @@ twig:
             - SimpleThingsFormExtraBundle:Form:field_type_help.html.twig
 ```
 
-You can also load the help extension form theme into your own form theme.
+Next, load the help extension form theme into your template.
```

Comment:
This is wrong. You are now sayign something totally different than the original message.